src/crush/CrushLocation.cc

```diff
@@ -48,6 +48,9 @@ int CrushLocation::update_from_hook()
   if (cct->_conf->crush_location_hook.length() == 0)
     return 0;
  
+#if defined(WITH_SEASTAR) && !defined(WITH_ALIEN)
+  ceph_abort_msg("crimson does not support crush_location_hook, it must stay empty");
```

Comment:
should we have a TODO comment/trello card for this so we don't forget it?

Reply:
Likely so. Rather not a high or even medium priority thing but already mentioned in the documentation: https://docs.ceph.com/en/octopus/rados/operations/crush-map/#custom-location-hooks.